modeling.py

```diff
@@ -245,37 +254,66 @@ def nan_to_none(x):
         statistics_rows['error_mean'].append(error_mean)
         absolute_error_mean = np.mean(np.abs(errors))
         statistics_rows['abs_error_mean'].append(absolute_error_mean)
-        variance = np.var(errors)
-        statistics_rows['variance'].append(variance)
+        std = np.std(errors)
+        statistics_rows['standard_deviation'].append(std)
         make_histogram(errors, key, False)
-        outliers_size = len(errors)
-        errors = [item for item in errors if abs(item) <= outlier]
-        outliers_size -= len(errors)
+
+        outlier_errors = errors.copy()
```

Comment:
nazwałbym to lowest_errors lub best_errors, bo outlier_errors sugeruje jakby tam były same wyniki odstające, zmienię przy najbliższej okazji, teraz merguję